README.md

```diff
@@ -0,0 +1,15 @@
+# learn-monitoring-troubleshooting
+Exercises component of the Monitoring &amp; Troubleshooting Data Streaming Applications course
+
+## Case Study 1: Monitoring Kafka Publishers in Python
```

Comment:
"Publishers" or "Producers" - just for the sake of consistency with the terms.

Reply:
Updated to producers; thanks for catching that!